lua/wikis/commons/MatchTicker/DisplayComponents/New.lua

```diff
@@ -231,6 +234,11 @@ function Details:tournament()
 
 	return mw.html.create('div')
 		:addClass('match-tournament')
+		:node(
+			self.displayGameIcons
+			and Game.icon{game=self.match.game, noLink=true, size='50px', spanClass='icon-small'}
+			or nil
+		)
 		:node(mw.html.create('div')
 			:addClass('tournament-icon')
 			:node(mw.html.create('div')
```

Comment:
Would it be possible to reuse `lua/wikis/commons/Widget/Tournament/Label.lua` instead?

Reply:
Ugh, the applied CSS is kind of different tho, would need to adjust classes to have specific content then

Reply:
Does the CSS need to be different though?